lib/samples/apply_colormap_renderer_to_raster/apply_colormap_renderer_to_raster.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_data.dart';
+import 'package:flutter/material.dart';
+import 'package:path_provider/path_provider.dart';
+
+class ApplyColormapRendererToRaster extends StatefulWidget {
+  const ApplyColormapRendererToRaster({super.key});
+
+  @override
+  State<ApplyColormapRendererToRaster> createState() =>
+      _ApplyColormapRendererToRasterState();
+}
+
+class _ApplyColormapRendererToRasterState
+    extends State<ApplyColormapRendererToRaster> {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        left: false,
+        right: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                  ),
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future<void> onMapViewReady() async {
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISImageryStandard);
```

Comment:
We should have our standard comment above this line, `// Create a map with an imagery basemap style.`